src/Header/Header.js

```diff
@@ -4,7 +4,7 @@ import './Header.css'
 const Header = () => {
   return (
     <header>
-      <img alt='Website Icon'/>
+			<img src='https://media.giphy.com/media/KXBtTtm3kB8BO/giphy.gif'alt='Website Icon'/>
```

Comment:
Pretty silly. Maybe on our own time we could research some more Actual logos?

Reply:
Agreed!!!